redis.conf

```diff
@@ -451,6 +451,15 @@ repl-disable-tcp-nodelay no
 # By default the priority is 100.
 replica-priority 100
 
+# Limits slave full synchronization speed.
+# The limit is enforced on the slave side.
+#
+# A value of 0 means no limit (that's default).
+#
+# For example to set limit to 100 Mbps use:
+#
+# slave-sync-speed-bps 100000000
```

Comment:
After this PR was closed, redis's maintainers changed the wording of setting names and comments to replica (e.g. replica-priority, min-replicas-max-lag) and made existing names aliases of the new names

It also looks like commits for new features (e.g. `implement FAILOVER command (8315)`) use replica for C variable names